### DISCUSSION TITLE
如何添加osmbuildings, 有封装吗

### DISCUSSION THREAD

**Discussion by kankantian** [zouyaoji/vue-cesium#480] [Q&A]
如何添加osmbuildings, 有封装吗

**Comment by zouyaoji** [comment]
试试这个组件 `VcPrimitiveOsmBuildings`

https://github.com/zouyaoji/vue-cesium/blob/dev/packages/components/primitives/osm-buildings/index.ts

**Reply by kankantian** [reply]
是不是文档没更新？ 在文档上没看到

**Reply by zouyaoji** [reply]
估计添加了忘记写上去了。。。 
`VcPrimitiveOsmBuildings` 直接放`vc-viewer` 或者  `vc-primitive-collection` 下面。

**Reply by kankantian** [reply]
好的 已安排上了
